overlay/ports/taglib/vcpkg.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "taglib",
+  "version-string": "1.12-20210216",
+  "port-version": 0,
```

```suggestion
  "version-string": "1.12",
```
I've never seen semver and the date combined like that. I don't know if vcpkg would handle it properly.

port-version is only needed if the port is updated without a new release of the upstream code

---

This is just a string without a meaning. The current versions uses this format.